NWEEI/NWEEI/Views/Home/Index.cshtml

```diff
@@ -19,3 +19,18 @@
     </p>
     <a asp-controller="Resources" asp-action="CareerMap">View the Green Buildings Career Map</a>
 </div>
+
+
+<div class="quick-section-container">
+    <!-- quick nweei news -->
+    <div class="quick-section">
+        <h3>NWEEI News</h3>
+        @await Component.InvokeAsync("QuickArticles", "NWEEI News")
```

Comment:
Brenda will love to see this. And you did it in a way that keeps the home page loading fast. Nice!